doc/testing.adoc

```diff
@@ -0,0 +1,114 @@
+# Testing
+
+This document will describe how we should try to write and maintain tests in Sirius Web.
+
+IMPORTANT: TLDR: Use backend integration tests if you don’t have large and complex pieces of frontend code to test that involve the user interface (diagram interactions, diagram layout, form description editor drag and drop, etc).
+They can easily be configured to tests very complex use cases involving domains, views, custom services, etc, they are very to execute fast and are stable.
```

Comment:
`they are very fast to execute and are stable` ?

Reply:
Done